babel.config.js

```diff
@@ -0,0 +1,18 @@
+const { BABEL_ENV, NODE_ENV } = process.env
+const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test'
+
+module.exports = {
+  presets: [['@babel/env', { modules: false, loose: true }], '@babel/react'],
+
+  plugins: [
+    cjs && '@babel/plugin-transform-modules-commonjs',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-object-rest-spread',
+    'babel-plugin-idx',
+    'annotate-pure-calls',
+    NODE_ENV !== 'test' && [
```

Comment:
This plugin was breaking my tests. I don't think it's useful to remove prop-types in tests so I have inserted this condition.